docs/hub/tutorial-add-library.md

```diff
@@ -0,0 +1,184 @@
+# Integrate your library with the Hub
+
+The Hugging Face Hub aims to facilitate the sharing of machine learning models, checkpoints, and artifacts. This endeavor includes integrating the Hub into many of the amazing third-party libraries in the community. Some of the incredible libraries already integrated include [spaCy](https://spacy.io/usage/projects#huggingface_hub), [AllenNLP](https://allennlp.org/), and [timm](https://rwightman.github.io/pytorch-image-models/), among many others. Integration means users can download and upload files to the Hub directly from your library. We hope you will integrate your library and join us in democratizing artificial intelligence for everyone!
+
+Integrating the Hub with your library provides many benefits, including:
+
+- Free model hosting for you and your users.
+- Built-in file versioning - even for huge files - made possible by [Git-LFS](https://git-lfs.github.com/).
+- All public models are powered by our lightning-fast [Inference API](https://api-inference.huggingface.co/docs/python/html/index.html).
```

Comment:
This is really optional imo. Also it's not really lightning-fast for 3rd party since we don't offer GPU and other than that the inference depends on the library integration

```suggestion
- All public models are powered by the [Inference API](https://api-inference.huggingface.co/docs/python/html/index.html).
```